dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/layers/Layers.scala

```diff
@@ -3,18 +3,39 @@ import me.shadaj.scalapy.tensorflow.keras.layers.{Layers => PyLayers}
 import me.shadaj.scalapy.tensorflow.api.scalaUtils.{PythonModule, PythonType}
 import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonOption._
 import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonUnion._
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonEnum
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonEnum._
 import scala.language.implicitConversions
 import me.shadaj.scalapy.numpy.NDArray
 import me.shadaj.scalapy.tensorflow.api.Tensor
 import me.shadaj.scalapy.tensorflow.api.Tensor._
 import me.shadaj.scalapy.py
 
+
+
+enum Padding(override private[api] val v: String) extends PythonEnum(v){
```

Comment:
Nit: do we have scalafmt for Dotty code?

Reply:
We will see :D